pkgs/tools/package-management/nix/common.nix

```
@@ -133,6 +133,9 @@ self = stdenv.mkDerivation {
   ];
 
   preConfigure =
+    ''
+      patchShebangs tests
```

```suggestion
      patchShebangs --build tests
```
and move to preCheck